dist/index.js

```diff
@@ -51,6 +52,13 @@ var _H = 'H'.charCodeAt(0);
 var _Y = 'Y'.charCodeAt(0);
 var _S = 's'.charCodeAt(0);
 
+function changeDpiBuffer(buffer, dpi, type) {
+  var headerChunk = new Uint8Array(buffer.slice(0, 33));
```

Comment:
once the first 33 bytes are converted to array, can we autoDetect the type? the png signature is fixed and the jpeg more or less too. In this way we can find out the type and having the function require an argument the other do not required.

Reply:
sure!

Reply:
added `getType` methods that check the relevant few header bytes.

Reply:
like this is the png start
`137 80 78 71 13 10 26 10`  <--- decimal values of the first 8 bytes.